fastlane/Fastfile

```diff
@@ -48,31 +48,34 @@ end
 
 desc 'Notarize the compiled binary'
 lane :notarize_binary do |_options|
-  notarize(
-    package: File.join(PROJECT_ROOT_FOLDER, 'out', 'Studio-darwin-x64', 'Studio.app'),
-    api_key_path: APPLE_API_KEY_PATH
-  )
-  notarize(
-    package: File.join(PROJECT_ROOT_FOLDER, 'out', 'Studio-darwin-arm64', 'Studio.app'),
-    api_key_path: APPLE_API_KEY_PATH
-  )
+   Dir[File.join(PROJECT_ROOT_FOLDER, 'out', '**', 'Studio.app')].each { |path|
+      notarize(
+        package: path,
+        api_key_path: APPLE_API_KEY_PATH
+      )
+   }
```

Comment:
👍 

Nitpick though: indentation has one extra space (3 instead of 2) 🙃

Reply:
Addressed in [850a66b](https://github.com/Automattic/studio/pull/85/commits/850a66b1f53d4ab719b35003f9c017ef316d088b)